### PR TITLE
docs: Add contribution guidelines

### PR DESCRIPTION
**Pull request by salonigupta1**
Fixes #29

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# CONTRIBUTING
+
+#### When contributing to this project, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
```
Comment (pnijhara) [inline contribution.md:3]: This should be a normal text and not a h4.

File: contribution.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+#### Please note we have a code of conduct, please follow it in all your interactions with the project.
```
Comment (pnijhara) [inline contribution.md:5]: Normal text.

File: contribution.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### PULL REQUEST PROCESS
+
+#### 1.Ensure any install or build dependencies are removed before the end of the layer when doing a build.
```
Comment (pnijhara) [inline contribution.md:9]: Normal point

File: contribution.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+#### 2.Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
```
Comment (pnijhara) [inline contribution.md:11]: Again this should be a normal text point.

File: contribution.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+### PULL REQUEST GUIDELINES
+
+#### 1.Use the present tense imperative mood in the example commit message.
+
+#### 2.Conventional Commits may not need to specify present tense, imperative mood for verbs. Follow the use of use the common present tense, imperative mood practice.
+
+#### 3.You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+
+#### [References](https://www.conventionalcommits.org/en/v1.0.0/)
```
Comment (pnijhara) [inline contribution.md:21]: Normal link.

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Comment by pnijhara** [comment]
@salonigupta1 you have written most of the content in h3 and h4. I recommend you to please go through the content and again and use the normal text wherever required. Also, make a directory named Development and add `contributing.md` and other files related to development process into this.

@syedjawadakhtar can you look into this, please?

**Comment by syedjawadakhtar** [comment]
- I guess @salonigupta1 is trying to make all the texts bold and bigger by putting it under h4. 
- Use headings only when there is a change in context. You may number the headings as 1. 2. 3. to reduce any complexity in the article. Similarly, for subheadings, use 1.1, 2.1 ...
- Follow the hierarchy for headings as well. h1 -> h2 -> h3 -> h4 for pretty printing.
- You can get a go-to guide for *markdown* here in this tutorial:
[https://github.com/luong-komorebi/Markdown-Tutorial](https://github.com/luong-komorebi/Markdown-Tutorial)

File: Development/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
+# CONTRIBUTING GUIDLINES
```
Comment (pnijhara) [inline Development/CONTRIBUTING.md:1]: This file should come under docs directory.
docs/Develoment/CONTRIBUTING.md

File: contribution.md (hunk 1 above)
Comment (pnijhara) [reply contribution.md:3]: Remove this file!

File: docs/CONTRIBUTING.md [hunk 7]
```diff
@@ -11,3 +11,13 @@ use GitHub pull requests for this purpose. Consult
 information on using pull requests.
 
 > Note : Every PR before require at least 2 approvals to get merged to the master branch.
+
```
Comment (pnijhara) [inline docs/CONTRIBUTING.md:14]: Remove this file also!

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Comment by pnijhara** [comment]
@salonigupta1 there are still changes required.
Since you have added a new subdirectory named Development. You can add this directory within docs directory and update nav link in mkdocs.yml file.

**Comment by pnijhara** [comment]
I am merging this PR. Will update the changes on my own!
Thank you for the contribution @salonigupta1

**Pull request merged**